docs/setup_guides/cloud.rst

```diff
@@ -100,45 +100,15 @@ Run the following from the commmand line
 
   gcloud auth login
 
-Now here is a bash script that will create a cluster corresponding to Pangeo
+A separate set of bash commands that will create a cluster corresponding to Pangeo
 need:
-- An incompressible default node pool for the Jupyterhub, web proxy, and user
-  notebook servers.
-- An auto scaling node pool for Dask workers.
 
-This script is availabe in pangeo/gce/setup-guide as ``1_create_cluster.sh``,
-so you can use it directly.
-
-.. code-block:: bash
-
-  #!/bin/bash
-
-  set -e
-
-  PROJECTID=<YOUR GOOGLE CLOUD PROJECT ID>
-  # this is the zone used by pangeo.pydata.org
-  ZONE='us-central1-b'
-
-  # cluster size settings: modify as needed to fit your needs / budget
-  NUM_NODES=2
-  MIN_WORKER_NODES=0
-  MAX_WORKER_NODES=100
-  CLUSTER_NAME='pangeo'
-
-  # https://cloud.google.com/compute/pricing
-  # change the machine typer based on your computing needs
-  WORKER_MACHINE_TYPE='n1-standard-4'
-
-  # create cluster on GCP
-  gcloud config set container/new_scopes_behavior true
-  gcloud config set project $PROJECTID
-  gcloud container clusters create $CLUSTER_NAME --num-nodes=$NUM_NODES --zone=$ZONE \
-      --machine-type=n1-standard-2 --no-enable-legacy-authorization
-  gcloud container node-pools create worker-pool --zone=$ZONE --cluster=$CLUSTER_NAME \
-      --machine-type=$WORKER_MACHINE_TYPE --preemptible --enable-autoscaling \
-      --num-nodes=$MIN_WORKER_NODES --max-nodes=$MAX_WORKER_NODES --min-nodes=$MIN_WORKER_NODES
-  gcloud container clusters get-credentials $CLUSTER_NAME --zone=$ZONE --project $PROJECTID
+  - An incompressible default node pool for the Jupyterhub, web proxy, and user
+    notebook servers.
+  - An auto scaling node pool for Dask workers.
 
+A script defining these is availabe in the pangeo/gce/setup-guide directory
+as `1_create_cluster.sh`_ and you can use that directly, after editing it.
```

Comment:
If I'm not mistaken, no need to edit the script, the only script that needs edition is the `gce-pangeo-environment.sh` one.